### PR TITLE
fix: vale check

### PR DESCRIPTION
**Pull request by RobPasMue**
Reverting #470 commited to main - if is failing. See https://github.com/ansys/actions/actions/runs/8833187453/job/24251985324

### CODE REVIEW

**Review by SMoraisAnsys — APPROVED** [review]
LGTM but I'm a bit confused with the changes and the current main branch.
Did afb1380 got pushed before this PR ? I may be missing something but they seem to be the same changes.

**Comment by RobPasMue** [comment]
The PR is not working properly. I reverted the commit and re-opened it. We need to fix the if statement.

File: doc-style/action.yml [hunk 1]
```diff
@@ -71,6 +71,37 @@ runs:
...
+      shell: bash
+      run: |
+        if [ $VALE_MAJOR_VERSION -ge 3 ]; then
+          if [ ! -d ./doc/styles/config/vocabularies ]; then
+            echo "Projects using Vale >=v3 require a doc/styles/config/vocabularies directory."
+            echo "See migration guide https://actions.docs.ansys.com/version/stable/migrations/docs-style-vale-version-update.html"
+            exit 1
+          fi
+        else
+          if [ ! -d ./doc/styles/Vocab ]
```
Comment (RobPasMue) [inline doc-style/action.yml:98]:
```suggestion
          if [ ! -d ./doc/styles/Vocab ]; then
```

File: doc-style/action.yml [hunk 2]
```diff
@@ -71,6 +71,37 @@ runs:
...
+          Ensure Vale configuration follows the right layout for the specified
+          version of Vale.
+
+    - name: "Check Vale's version"
+      shell: bash
+      run: |
+        version=${{ inputs.vale-version }}
+        vale_major_version=$( echo $version | cut -d '.' -f 1)
+        echo "VALE_MAJOR_VERSION=$(echo $major)" >> $GITHUB_ENV
+
```
Comment (RobPasMue) [inline doc-style/action.yml:87]:
```suggestion
        echo "VALE_MAJOR_VERSION=$(echo $major)" >> $GITHUB_ENV
        echo "VALE_MAJOR_VERSION set to $vale_major_version"  # Add this line for debugging

```

File: doc-style/action.yml [hunk 3]
```diff
@@ -71,6 +71,37 @@ runs:
...
+    - name: "Verify Vale's configuration layout according to selected version"
+      shell: bash
+      run: |
+        if [ $VALE_MAJOR_VERSION -ge 3 ]; then
```
Comment (RobPasMue) [inline doc-style/action.yml:91]:
```suggestion
        if [ -z "$VALE_MAJOR_VERSION" ]; then
          echo "VALE_MAJOR_VERSION is not set."
          exit 1
        fi

        if [ $VALE_MAJOR_VERSION -ge 3 ]; then
```

**Comment by RobPasMue** [comment]
Okay - now it works like a charm. Testing performed here: https://github.com/ansys/pyansys-geometry/pull/1168

**Pull request merged**